wagtail/wagtailadmin/widgets.py

```diff
@@ -137,6 +137,8 @@ class AdminPageChooser(AdminChooser):
     def __init__(self, target_models=None, can_choose_root=False, **kwargs):
         super(AdminPageChooser, self).__init__(**kwargs)
 
+        if target_models:
+            self.choose_one_text += ' (' + ', '.join([model._meta.model_name for model in target_models]) + ')'
```

Comment:
Could you use `verbose_name` instead of `model_name`, please? :)
I think `model_name` is not localized.